packages/patternlab/styleguide/source/assets/js/modules/inlineOverlay.js

```diff
@@ -6,6 +6,9 @@ export default (function (window,document,$,undefined) {
   let toggleClass = '.js-inline-overlay-toggle';
   let titleClass  = '.js-inline-overlay-title';
   let feedbackButton = $('body').find('.ma__fixed-feedback-button');
+  let tocFocusableElements = $('.js-inline-overlay').find(':focusable');
```

Comment:
For the TOC part, can you walk me over what needs to be fixed? And how to test it?